From 7f69b462e8ccc5949944e5d1d71d8268255016e2 Mon Sep 17 00:00:00 2001
From: Drew Moseley <drew@moseleynet.net>
Date: Fri, 11 Nov 2022 11:23:30 -0500
Subject: [PATCH] apalis-imx8: Use mender_dtb_name for fdtfile.

We use preboot to unconditionally set the value of fdtfile to
mender_dtb_name.  This is more consistent with how Mender is
expected to handle DTB names when using Yocto.

Changelog: Title
Signed-off-by: Drew Moseley <drew@moseleynet.net>
---
 configs/apalis-imx8_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/apalis-imx8_defconfig b/configs/apalis-imx8_defconfig
index 173a445233..382472efcf 100644
--- a/configs/apalis-imx8_defconfig
+++ b/configs/apalis-imx8_defconfig
@@ -23,7 +25,7 @@ CONFIG_FIT_VERBOSE=y
 CONFIG_OF_SYSTEM_SETUP=y
 CONFIG_BOOTDELAY=1
 CONFIG_USE_PREBOOT=y
-CONFIG_PREBOOT="test -n ${fdtfile} || setenv fdtfile ${soc}-apalis${variant}-${fdt_board}.dtb"
+CONFIG_PREBOOT="setenv fdtfile ${mender_dtb_name}"
 CONFIG_LOG=y
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_DISPLAY_BOARDINFO_LATE=y
-- 
2.38.1

